### PR TITLE
(IMAGES-490) Packer Acceptance Test Environment

### PR DESCRIPTION
**Pull request by jcoconnor**
Setup an acceptance test structure for Packer Builds.
This is tied to jenkins JJB definitions associated with this ticket.

### CODE REVIEW

**Comment by jcoconnor** [comment]
@puppetlabs/agent-team This is ready for review now.

Test run has successfully cloned a new ```win-2012r2-x86_64``` template and run the ```test.rb``` script against it. See https://jenkins-imaging.delivery.puppetlabs.net/view/Windows/job/imaging_windows-2012r2_x86_64_vmware_vsphere.cygwin_acceptance_packer/9/console

File: acceptance/Gemfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  if place =~ /^(git:[^#]*)#(.*)/
+    [fake_version, { :git => $1, :branch => $2, :require => false }].compact
+  elsif place =~ /^file:\/\/(.*)/
+    ['>= 0', { :path => File.expand_path($1), :require => false }]
+  else
+    [place, { :require => false }]
+  end
+end
+
+gem "beaker", *location_for(ENV['BEAKER_VERSION'] || '~> 3.1.0')
```
Comment (Magisus) [inline acceptance/Gemfile]: In all our other project we have recently dropped the 'z' version from this specification, to allow beaker to update to the most recent 'y' version. Is there a reason you need to be pinned to 3.1 for this, or can you use the most recent 'y' version?

Comment (jcoconnor) [inline acceptance/Gemfile]: Fixed

**Comment by ScottGarman** [comment]
:+1:

**Pull request merged**